modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -139,6 +139,9 @@ export default class I3SConverter {
     meshTopologyTypes: new Set(),
     metadataClasses: new Set()
   };
+  tileCountTotal: number = 0;
```

Comment:
Nit: Add a TSDoc in front of each field. Then when hovering over these in vscode the documentation is shown.
```suggestion
  /** Total count of tiles in tileset */
  tileCountTotal: number = 0;
```

Reply:
Done